internal/client/valve/replay.go

```diff
@@ -36,21 +43,19 @@ func NewReplay(httpClient *http.Client) *Replay {
 // Based on that value, we can download the replay DotA2 file directly using that URL.
 //
 // destination parameter specifies the file destination to save the downloaded replay file.
-func (r *Replay) Download(ctx context.Context, replayURL string, destination string) error {
-	fileName := getFileName(replayURL)
+func (r *Replay) Download(ctx context.Context, url, dir string) error {
+	fileName := path.Base(url)
```

Comment:
replaced `getFilename` func and hence test file related to it

Reply:
TIL, that `path.Base()` will return the last element of path